.storybook/preview.js

```diff
@@ -6,7 +6,11 @@ export const parameters = {
       date: /Date$/,
     },
   },
-  // Notifies Chromatic to pause the animations when they finish at a global level
-  // This is because Chromatic doesn't show the cover image when the animation starts with opacity 0
-  chromatic: { pauseAnimationAtEnd: true },
+  chromatic: {
+    // Notifies Chromatic to pause the animations when they finish at a global level
+    // This is because Chromatic doesn't show the cover image when the animation starts with opacity 0
+    pauseAnimationAtEnd: true,
+    // Set the viewports that Chromatic tests with globally.
```

Comment:
```suggestion
    // Set the viewport widths that Chromatic tests with globally.
```

Reply:
Please consider if we have any reference for the values we use.

When we did not specify it we have the Chromatic default, whatever that was. Now we use 400 and 1200. Are these values based on two reference devices, do they have a relation to our breakpoints or [the devices Storybook will resize to](https://storybook.js.org/docs/react/essentials/viewport).